app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -114,18 +115,6 @@ class NavigationDrawerTestActivityTest {
     }
   }
 
-  @Test
-  fun testNavigationDrawerTestActivity_clickNavigationDrawerHamburger_clickOnHeader_opensProfileProgressActivity() {
```

Comment:
Why this testcase was removed.

Reply:
Actually this has been shifted to `HomeActivity` tests

Reply:
Removed unused import.